pkg/skaffold/deploy/helm/helm.go

```diff
@@ -469,7 +477,7 @@ func (h *Deployer) deployRelease(ctx context.Context, out io.Writer, r latest.He
 
 // getRelease confirms that a release is visible to helm
 func (h *Deployer) getRelease(ctx context.Context, helmVersion semver.Version, releaseName string, namespace string) (bytes.Buffer, error) {
-	// Retry, because under Helm 2, at least, a release may not be immediately visible
+	// Retry, because sometimes a release may not be immediately visible
```

Comment:
from the comments, looks like the retry logic was explicitly added maybe for helm2. Is there a way to verify if this is needed for helm3?

Reply:
thought about it, but it can't hurt to leave i think. if it's not needed then it will just get bypassed, and it serves as a safeguard in case we do actually end up needing it.